auto-value-parcel/src/test/java/com/ryanharter/auto/value/parcel/AutoValueParcelExtensionTest.java

```diff
@@ -467,6 +469,224 @@
         .generatesSources(expectedNotMatching, expectedMatching);
   }
 
```

Comment:
The following two tests are copied from the format of the existing test 'describeContentsOmittedWhenAlreadyDefined'